test/helpers/fakeGrpcTransport.ts

```diff
@@ -208,8 +207,7 @@ export class StubTransportBuilder {
   }
 }
 
-export interface TriggerableTransport {
-  (options: TransportOptions): Transport;
+export interface TriggerableTransport extends grpc.TransportFactory {
```

Comment:
This change doesn't look like it should belong in this PR - did something go wrong whilst rebasing / merging in the changes from master?

Reply:
Yeah, that's my bad. Fixed.